cmd/pmm-configurator/update.go

```diff
@@ -134,9 +137,16 @@ func returnLog(w http.ResponseWriter, req *http.Request, timestamp string, httpS
 		return
 	}
 
-	updateState := "finished"
+	var updateState string
 	if isPidAlive(pidInt) {
-		updateState = "in-progress"
+		updateState = "running"
+	} else {
+		re = regexp.MustCompile(`localhost .* failed=0\s`)
```

Comment:
Since the regex is a constant, can we convert it to a pgk var to avoid compiling the regex every time we run this func